plugin/trino-raptor-legacy/src/main/java/io/trino/plugin/raptor/legacy/storage/OrcFileWriter.java

```diff
@@ -217,10 +216,10 @@ private static OrcSerde createSerializer(Properties properties)
 
     private static RecordWriter createRecordWriter(Path target, List<Long> columnIds, List<Type> columnTypes, boolean writeMetadata)
     {
-        try (FileSystem fileSystem = new SyncingFileSystem(CONFIGURATION)) {
```

Comment:
Do we want to change the SyncingFileSystem usage? This actually seemed fine before. For example, if you disable this, then you break the deleteOnExit behavior if that is being used.

Reply:
It wasn't necessary to close before, as we didn't use that behavior. It seems better to simply remove the close than to suppress the warning.